### ISSUE TITLE
Missing file?

### ISSUE THREAD

**Issue by takikawa** [thinkmoore/bounded#1]
When I try to install this, I get the following error:

```
/home/asumu/plt/racket-git/racket/share/pkgs/bounded/private/bounded.rkt:4:9: cannot open module file
  module path: #<path:/home/asumu/plt/racket-git/racket/share/pkgs/bounded/private/bounded-impersonator-util.rkt>
  path: /home/asumu/plt/racket-git/racket/share/pkgs/bounded/private/bounded-impersonator-util.rkt
  system error: No such file or directory; errno=2
  context...:
   standard-module-name-resolver
   standard-module-name-resolver
   standard-module-name-resolver
   standard-module-name-resolver
   /home/asumu/plt/racket-git/pkgs/racket-index/setup/scribble.rkt:1513:27
   /home/asumu/plt/racket-git/pkgs/racket-index/setup/scribble.rkt:901:0: load-doc/ensure-prefix
   /home/asumu/plt/racket-git/pkgs/racket-index/setup/scribble.rkt:1155:13
   .../parallel-do.rkt:390:17
   /home/asumu/plt/racket-git/racket/collects/setup/../racket/private/more-scheme.rkt:261:28
   /home/asumu/plt/racket-git/racket/collects/setup/parallel-do.rkt:435:20: loop

  context...:
   /home/asumu/plt/racket-git/racket/collects/setup/parallel-do.rkt:334:4: work-done method in list-queue%
   /home/asumu/plt/racket-git/racket/collects/setup/parallel-do.rkt:284:17
   /home/asumu/plt/racket-git/racket/collects/setup/parallel-do.rkt:238:4
   /home/asumu/plt/racket-git/pkgs/racket-index/setup/scribble.rkt:138:0: setup-scribblings
   /home/asumu/plt/racket-git/racket/collects/setup/setup-core.rkt:70:0: setup-core
   /home/asumu/plt/racket-git/racket/collects/setup/setup.rkt:65:3
   /home/asumu/plt/racket-git/racket/collects/pkg/main.rkt:17:0: setup
   (submod /home/asumu/plt/racket-git/racket/collects/pkg/main.rkt main): [running body]
   /home/asumu/plt/racket-git/racket/collects/pkg/raco.rkt: [traversing imports]
```

I suspect that module is missing. Steps to reproduce: `raco pkg install https://github.com/thinkmoore/bounded.git`

**Comment by takikawa** [comment]
Whoops, I guess this module is supposed to be built by the `build.sh` script? My fault for not reading the instructions.

**Comment by thinkmoore** [comment]
Yes, it needs to be built with build.sh. I've also never tried actually installing it, so I'm not sure if that works properly...

**Comment by takikawa** [comment]
It doesn't quite work for me but I'm not sure why. I'll open another issue if I figure it out. Also, I think it's possible to set it up to automatically build the C part on setup (example: https://github.com/tonyg/racket-ansi/blob/master/ansi/private/install.rkt) but then you have to add `schpriv.h` to the include paths.

**Comment by thinkmoore** [comment]
Thanks for the pointer to Tony's project. I think I can get it working based on that example.

**Comment by thinkmoore** [comment]
Fixed in e062d5a! Let me know if it works for you.

**Comment by takikawa** [comment]
Yep, `raco setup` now works! Thanks.
